bin/mkosi

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/bash
+
+PYTHON=$(command -v python3)
+PYVERSION=$($PYTHON -c '
```

Comment:
`python --version | cut -d ' ' -f 2 | cut -d '.' -f 1-2`?

Reply:
True, that is a reasonable solution. Alternatively 
```bash
python --version | awk '{ split($2, a, "."); print a[1] "." a[2] }'
```
which omits starting another process, but I guess I'll go with your solution as to not depend on AWK.